finance.md

```diff
@@ -0,0 +1,69 @@
+1. Generate a team timesheet in Clockify and save (e.g. [aether_201908](https://clockify.me/bookmarks/5dab94a0ae125662897c69b4))
+1. Export team timesheet as PDF to be sent to client, if applicable
+
+This process is done using the Clockify [Summary Report](https://clockify.me/reports/summary). Saved reports are found under [Saved Reports](https://clockify.me/reports/saved).
+
+### Generating invoices
+
+1. Generate an invoice in **Quickbooks Online** based on the pricing structure (e.g. timesheet hours and project rate)
+1. Add operational cost line item, if applicable
+1. Add GST, if applicable
+
+The general guidance is that the client is responsible for paying applicable GST and transaction costs charged by their financial institution, and Hypha absorbs the cost of receiving the money (e.g. cost of receiving a wire).
+
+Invoices (and timesheets, if requested) are sent to clients and are payable in CAD, USD, or EUR. Invoices are due 14 days after the invoice date, after which a 5% late fee will be charged for each additional week.
+
+## Member payroll
+
+Member salaries are paid monthly and vary month-to-month depending on their billable hours of work. At the end of each month, each Member's entitled pay is calculated based on the projects and roles they participate in. After withholding taxes and other applicable deductions (e.g. CPP and EI), the net amount is electronically tranferred to the Member's bank account.
+
+Expense reimbursements, which require photos uploaded to Quickbooks Online, are paid out monthly with the salary amount as non-taxable items in the paycheque.
+
+At the end of each tax year, each Member is entitled to a vacation amount that equals 4% of their salary for the year. This amount will be added to the salary for that month.
+
+All payroll, expense reimbursements, and withheld taxes are managed using Quickbooks Online, which is linked to our main Desjardins account.
+
+### Calculating member salary
+
+Each Member's pay from billable project work will have a percent-based portion taken from it to fund administrative costs and collective programs. Collective programs may include non-billable projects, member health benefits, CPP & EI, "rainy days" reserve contributions, as well as and other collective expenses of the co-operative. The amount is calculated based on the invoiced amount before transaction costs. The exact percentage is the same across all projects during a period and can be revised as necessary by consensus.
+
+### Reporting and remitting taxes
+
+Hypha has the following accounts with the Canada Revenue Agency (CRA):
+
+- **RC** for corporate income tax, which is reported and remitted each year
+- **RP** for payroll deduction, which is reported and remitted each month
+
+As a non-profit organization, our co-operative should have tax-exempt status and pay no corporate income tax. We do not have to collect GST/HST from clients and get GST/HST refunds for our puchases.
```

Comment:
This isn't my understanding. We don't need to collect HST for mission work (like education or workshops) but **we _do_ need to collect for traditional web dev work for traditional clients**. (Though our sub-$30,000 general threshold for HST means we can avoid it for a bit.).

This information is from Matt @ 4th Pig, who says their construction company 100% has to collect HST for home builds, but don't need to collect HST for some grant-supported work, or for education offerings they make.

I feel like we should default to collecting HST (after we're over over $30,000 threshold) until we have more certainty about what the "mission-based" carve-outs are. This might require convo with Fourth Pig.

Reply:
Yes this maps to my understanding we'll need to collect HST